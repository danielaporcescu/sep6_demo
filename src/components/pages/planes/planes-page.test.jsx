import React from 'react';

import Enzyme, { shallow, render, mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import PlanesPage from './planes-page';

Enzyme.configure({ adapter: new Adapter() })

it('Renders correctly FlightsPage', () => {
    const wrapper = shallow(<PlanesPage />)
  
    expect(toJson(wrapper)).toMatchSnapshot();
  });