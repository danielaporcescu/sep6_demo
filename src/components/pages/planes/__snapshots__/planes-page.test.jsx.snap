// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly FlightsPage 1`] = `
<div
  className="makeStyles-root-1"
>
  <WithStyles(ForwardRef(Grid))
    container={true}
    justify="space-around"
    spacing={2}
  >
    <WithStyles(ForwardRef(Grid))
      item={true}
      md={6}
    >
      <WithStyles(ForwardRef(Paper))
        className="makeStyles-paper-2"
      >
        <Styled(MuiBox)
          paddingBottom={2}
        >
          The manufacturers that have more than 200 planes
        </Styled(MuiBox)>
        <Loader />
      </WithStyles(ForwardRef(Paper))>
    </WithStyles(ForwardRef(Grid))>
    <WithStyles(ForwardRef(Grid))
      item={true}
      md={6}
    >
      <WithStyles(ForwardRef(Paper))
        className="makeStyles-paper-2"
      >
        <Styled(MuiBox)
          paddingBottom={2}
        >
          The number of flights each manufacturer with more than 200 planes are responsible for
        </Styled(MuiBox)>
        <Loader />
      </WithStyles(ForwardRef(Paper))>
    </WithStyles(ForwardRef(Grid))>
    <WithStyles(ForwardRef(Grid))
      item={true}
      md={12}
    >
      <WithStyles(ForwardRef(Paper))
        className="makeStyles-paper-2"
      >
        <Styled(MuiBox)
          paddingBottom={2}
        >
          The number of planes of each Airbus Model
        </Styled(MuiBox)>
        <Loader />
      </WithStyles(ForwardRef(Paper))>
    </WithStyles(ForwardRef(Grid))>
  </WithStyles(ForwardRef(Grid))>
</div>
`;
